_data/steps.yml

```diff
@@ -20,8 +20,7 @@
 
 - title: Deploy an end-to-end architecture
   description: |
-    You can assemble all the code from the IaC Library yourself by following our
-    <a href="/guides">Production Deployment Guides</a>, or you can have the Gruntwork team assemble a
+    You can assemble all the code from the IaC Library yourself or you can have the Gruntwork team assemble a
```

Comment:
Perhaps update the link to the docs site? 

"You can assemble all the code from the IaC Library yourself by following our docs or you can have..."